tests/test_utils.py

```diff
@@ -928,99 +929,24 @@ def test_static_filtering():
     ground_nsrts = (set(utils.all_ground_nsrts(nsrt1, objects))
                     | set(utils.all_ground_nsrts(nsrt2, objects)))
     assert len(ground_nsrts) == 8
-    atoms = {
+    atoms = {pred1([cup1, plate1]), pred1([cup1, plate2])}
+    reachable_atoms = utils.get_reachable_atoms(ground_nsrts, atoms)
+    assert {
         pred1([cup1, plate1]),
         pred1([cup1, plate2]),
         pred2([cup1, plate1]),
-        pred2([cup1, plate2]),
+        pred2([cup1, plate2])
+    }.issubset(reachable_atoms)
+    assert reachable_atoms & {
```

Comment:
What's up with this test, why can't you just check whatever you think reachable_atoms should be with a `==`?